index.html

```diff
@@ -5,242 +5,241 @@
+                    <div class="about-skill-heading">
+                        <h4 class="about-heading-small">Skills</h4>
+                        <i class="fa-solid fa-angle-right"></i>
+                    </div>
+                </li>
+                <hr>
+            </ul>
         </div>
-
-
-        <div class="skillset">
-            <div class="languages">
-                <div class="languages-heading">
-                    <span class="language-title">LANGUAGES</span>
-                    <button class="chevron-button">
-                        <img src="./Img/chevron-down.png" alt="" class="chevron">
-                    </button>
-                </div>
-                <ul class="language-list">
-                    <li><img src="./Img/JS-icon.png" alt="" class="language-icon">
-                        <span class="language-name">JavaScript</span>
-                    </li>
-
-                    <li><img src="./Img/HTML5-icon.png" alt="" class="language-icon">
-                        <span class="language-name">HTML</span>
-                    </li>
-
-                    <li><img src="./Img/CSS-icon.png" alt="" class="language-icon">
-                        <span class="language-name">CSS</span>
-                    </li>
-
-
-                </ul>
-            </div>
-
-            <div class="frameworks">
-                <span class="frameworks-title title">Frameworks</span>
-                <button class="chevron-button">
-                    <img src="./Img/chevron-right.png" alt="" class="chevron">
-                </button>
-            </div>
-            <div class="skills">
-                <span class="skills-title title">Skills</span>
-                <button class="chevron-button">
-                    <img src="./Img/chevron-right.png" alt="" class="chevron">
-                </button>
-            </div>
+        <!-- Contact Me Section Starts Here -->
+        <div class="contact-me-form-container">
+            <h1 class="contact-me-form-heading">Contact me</h1>
+            <p class="contact-me-form-paragraph">If you have an application you are interested in developing, a
+                feature that you need built
+                or a project that
+                needs coding. I'd love to help with it</p>
+            <form action="https://formspree.io/f/mqkoaqjr" method="POST" class="contact-me-form">
+                <input class="contact-me-form-text-input" required maxlength="30" name="name" type="text"
+                    placeholder="Enter Your Name">
+                <input class="contact-me-form-text-input" required name="email" type="email"
+                    placeholder="Enter Your Email">
+                <textarea class="contact-me-form-text-input" required maxlength="500" name="message" cols="30" rows="9"
+                    placeholder="Write your message here"></textarea>
+                <button class="contact-me-form-button" type="submit">Get In Touch</button>
+            </form>
```

Comment:
- [ ] You stick to the Figma design as much as you could :+1: and that almost okay :smile: , but you should make sure your design sticks to the Figma design as much as possible :smile:. Kindly resize this form and contact form section to meet the requirements :100: 

| Your design |  Figma design |
| :---: | :---: |
| ![Screenshot from 2023-01-22 01-28-48](https://user-images.githubusercontent.com/80458199/213894397-3ca4682f-da49-4a82-9e07-8ee6453f4b35.png) | ![Screenshot from 2023-01-22 01-44-34](https://user-images.githubusercontent.com/80458199/213894401-a78960ad-57e7-471a-83c2-6d80248f2648.png) |